engines/sword1/sword1.cpp

```diff
@@ -772,4 +852,67 @@ void SwordEngine::reinitRes() {
 	_screen->draw();
 }
 
+void SwordEngine::updateTopMenu() {
+	_menu->refresh(MENU_TOP);
+}
+
+void SwordEngine::updateBottomMenu() {
+	_menu->refresh(MENU_BOT);
+}
+
+void SwordEngine::fadePaletteStep() {
+	_screen->fadePalette();
+}
+
+void SwordEngine::startFadePaletteDown(int speed) {
+	_screen->startFadePaletteDown(speed);
+
+	// Fade audio here
```

Comment:
I'd suggest writing `TODO: Fade audio here`, the same below.

Reply:
Sure!